.github/actions/send-warning-via-slack/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Send Warning Notification via Slack'
+description: 'Sends a warning notification via Slack'
+inputs:
+  message:
+    description: 'The message to include in the Slack notification'
+    required: true
+    type: string
+runs:
+  using: "composite"
```

Comment:
why composite action and not a workflow call?

Reply:
ahh ok, I am seeing that we have already composite actions, I missed that we started to add them